ow_lander/launch/planning.launch

```diff
@@ -23,7 +23,7 @@
     </include>
 
     <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
-      <param name="use_gui" value="$(arg use_gui)"/>
+      <param name="use_gui" value="$(arg use_gui)"/> <!-- deprecated param -->
```

Comment:
What's the reason for deprecating use_gui? And is there a ticket to actually remove it from our files?

Reply:
You'll have to ask OSRF about their reasons
http://wiki.ros.org/noetic/Migration#Joint_State_Publisher_GUI

I did not create a ticket for that, but I suppose I should. What uses planning.launch and rviz.launch, if anything? I would've changed these to launch the now separate joint_state_publisher_gui node, but I wasn't sure how to test the change without knowing where and how the files are used.

Reply:
I only left the param in because the following pages says it will still be honored in noetic, but this actually contradicts what the Migration page says.
http://wiki.ros.org/joint_state_publisher#GUI

Reply:
It appears planning.launch and rviz.launch are no longer used. We should discuss this with @anjan1984 and try to do some cleanup.